manim/mobject/graph.py

```diff
@@ -338,6 +380,68 @@ def update_edge(e, u=u, v=v):
             update_edge(edge)
             edge.add_updater(update_edge)
 
+    def _tree_layout(
```

Comment:
I don't think that `_tree_layout` should be a class method; IMO it can simply be a module-level function in `graph.py` (especially because you don't need to use `self` anywhere).